### PR TITLE
fix: change src image to configuration

### PR DESCRIPTION
**Pull request by AuraAlba**
### Ticket
https://agile-jira.pearson.com/browse/PADV-903

### Description
Change scr logo to get from configurations

### Changes made
Change src logo

### How to test
Clone the Institution Portal MFE
Run npm install.
Run npm run start
Go to http://localhost:1980/

### CODE REVIEW

File: src/features/Main/Header/index.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const Header = () => {
     <header className="institution-header py-4 px-3">
       <div className="header-left d-flex">
         <a href={`${getConfig().LMS_BASE_URL}`}>
-          <img src="/images/logoPearson.png" alt="icon" />
+          <img src={`${getConfig().LOGO_URL_INSTITUTION}`} alt="icon" />
```
Comment (anfbermudezme) [inline src/features/Main/Header/index.jsx:16]: What do you think?

```suggestion
          <img src={`${getConfig().LOGO_INSTITUTION_PORTAL}`} alt="icon" />
```

File: src/features/Main/Header/index.scss [hunk 2]
```diff
@@ -27,7 +27,7 @@ header.institution-header {
   }
 
   .header-left img {
-    max-width: 30px;
+    height: 1.8rem;
```
Comment (anfbermudezme) [inline src/features/Main/Header/index.scss:30]: Why this change?

**Review by anfbermudezme — CHANGES_REQUESTED** [review]

File: src/features/Main/Header/index.scss (hunk 2 above)
Comment (AuraAlba) [reply src/features/Main/Header/index.scss:30]: the width can change if logo is longer or not, but the height should be fixed to avoid be bigger than the other elements

**Review by anfbermudezme — APPROVED** [review]
